src/sonic-yang-models/yang-models/sonic-bgp-common.yang

```diff
@@ -0,0 +1,540 @@
+            type boolean;
+            description "Connected checked disabled";
+            sonic-ext:custom-validation ValidateDisableConnectedCheck;
+        }
+
+        leaf enforce_first_as {
+            type boolean;
+            description "Enforce first AS for EBGP";
+        }
+
+        leaf solo_peer {
+            type boolean;
+            description "Part of own update group";
+        }
+
+        leaf ttl_security_hops {
+            type uint8 {
+                range 1..254;
+            }
+            must "((current() = 1 and (starts-with(current()/../neighbor, 'Eth') or starts-with(current()/../neighbor, 'Po') or starts-with(current()/../neighbor, 'Vlan'))) or (not(starts-with(current()/../neighbor, 'Eth') or starts-with(current()/../neighbor, 'Po') or starts-with(current()/../neighbor, 'Vlan'))))" {
+                error-message "Hops for connected neighbor cannot exceed 1";
+            }
+            description "TTL";
+        }
+
+        leaf bfd {
+            type boolean;
+            description "BFD status";
+        }
+
+        leaf bfd_check_ctrl_plane_failure {
+            type boolean;
+            description "Link dataplane status with BGP control plane";
+        }
+
+        leaf capability_dynamic {
+            type boolean;
+            description "Advertise dynamic capability";
+        }
+
+        leaf dont_negotiate_capability {
+            type boolean;
+            description "Do not perform capability negotiation";
+        }
+
+        leaf enforce_multihop {
+            type boolean;
+            description "Enforce EBGP neighbors perform multihop";
+        }
+
+        leaf override_capability {
+            type boolean;
+            description "Override capability negotiation result";
+        }
+
+        leaf peer_port {
+            type uint16;
+            description "Peer port number";
+        }
+
+        leaf shutdown_message {
+            type string {
+                length "1..127";
+            }
+            description "Message on peer/Neighbor shutdown";
+        }
+
+        leaf strict_capability_match {
+            type boolean;
+            sonic-ext:custom-validation ValidateStrictAndOverRideCapability;
+            description "Strict capability negotiation match";
+        }
+
+        leaf admin_status {
+            type boolean;
+            description "To enable BGP peer";
+        }
+
+        leaf local_as_no_prepend {
+            type boolean;
+            description "Do not prepend local-as to updates from ebgp peers";
+        }
+
+        leaf local_as_replace_as {
+            type boolean;
+            description "Do not prepend local-as to updates from ibgp peers";
+        }
+    }
+
+
+    grouping sonic-bgp-cmn-af {
+
+        leaf afi_safi {
+            type string;
+            description "Address family";
+        }
+
+        leaf admin_status {
+            type boolean;
+            description "Enabled status";
```

Comment:
Please change the description tom indicate that this reflects address family active/inactive

Reply:
sure